webviews/src/views/RokuAutomationView/ConfigControls.svelte

```diff
@@ -0,0 +1,141 @@
+<script>
+    export let selectedConfig; // string
+    export let configs; // array of configs
+
+    let showModal;
+    let inputValue = '';
+    let _inputValue = inputValue;
+    let dialog;
+
+    $: if (inputValue !== '') {
+        let found = configs.find((c) => c.name === inputValue);
+        if (!found) configs.push({ name: inputValue, steps: [] });
+        selectedConfig = inputValue;
+        inputValue = '';
+        configs = configs;
+    }
+
+    const pressOK = () => {
+        if (typeof _inputValue === 'string' && _inputValue !== '') {
+            inputValue = _inputValue;
+            // @ts-ignore
+            _inputValue = '';
+            console.log(`***** Pressed OK and inputValue=${inputValue}`);
+            showModal = false;
+            dialog.close();
+        }
+    };
+
+    $: if (dialog && showModal) dialog.showModal();
+
+    function deleteConfig() {
+        configs = configs.filter((c) => c.name !== selectedConfig);
+        selectedConfig = configs[0].name;
+    }
+</script>
+
+<style>
+    .button-group {
+        display: inline;
+    }
+    .btn {
+        border: none;
+        color: black;
+        border: 1px solid lightgray;
+        font-size: 16px;
+        cursor: pointer;
+    }
+
+    .btn:hover {
+        background-color: RoyalBlue;
+    }
+    dialog {
+        max-width: 32em;
+        border-radius: 0.2em;
+        border: none;
+        padding: 0;
+    }
+    dialog::backdrop {
+        background: rgba(0, 0, 0, 0.3);
+    }
+    dialog > div {
+        padding: 1em;
+    }
+    dialog[open] {
+        animation: zoom 0.3s cubic-bezier(0.34, 1.56, 0.64, 1);
+    }
+    @keyframes zoom {
+        from {
+            transform: scale(0.95);
+        }
+        to {
+            transform: scale(1);
+        }
+    }
+    dialog[open]::backdrop {
+        animation: fade 0.2s ease-out;
+    }
+    @keyframes fade {
+        from {
+            opacity: 0;
+        }
+        to {
+            opacity: 1;
+        }
+    }
+    button {
+        display: inline;
+    }
+</style>
+
+<link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
```

Comment:
We already have an icons library available, so let's remove this dynamic web import in favor of those. 

https://github.com/rokucommunity/vscode-brightscript-language/blob/923a3178a90a7f37b29d9b897c1cbb311c3a9eb4/webviews/src/views/RokuFileSystemView/RokuFileSystemView.svelte#L5

Here's the list of all icons available. https://code.visualstudio.com/api/references/icons-in-labels#icon-listing

Reply:
yep - in the process of polishing (one reason why I added as a draft, so you could review the functionality while I clean it up)